### PR TITLE
Initial version of CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by aaltat**

### CODE REVIEW

**Comment by just-be-dev** [comment]
Can we perhaps make this a .md file so that it's more legible from github?

**Comment by aaltat** [comment]
Sure thing. Something more or should it be put in the master and update the wiki pages?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+=======================
+
+These guidelines instruct how to submit issues and contribute code to the
+`Selenium2Library project <https://github.com/robotframework/Selenium2Library>`_.
+Other great ways to contribute include answering questions and participating discussion on `robotframework-users
+<https://groups.google.com/forum/#!forum/robotframework-users>`_ mailing list
+and other forums as well as spreading the word about the framework one way or
+the other.
+
+.. contents::
```
Comment (just-be-dev) [inline CONTRIBUTING.md]: This isn't needed anymore

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
+Contribution guidelines
+=======================
+
+These guidelines instruct how to submit issues and contribute code to the
+`Selenium2Library project <https://github.com/robotframework/Selenium2Library>`_.
+Other great ways to contribute include answering questions and participating discussion on `robotframework-users
```
Comment (just-be-dev) [inline CONTRIBUTING.md]: markdown links are formatted a little differently. Use `[link](url)`.

**Comment by pekkaklarck** [comment]
How is md more legit than rst at GitHub in this context? I'm fine either way.

**Comment by just-be-dev** [comment]
Well, I suppose saying github was a misnomer in that context. It's easier (for me personally) to read through unrendered markdown than it is rst. It doesn't necessarily matter, but I'd rather use markdown.

**Comment by aaltat** [comment]
I don't feel strongly on the matter, I am happy in both ways.

**Comment by aaltat** [comment]
Fixed some of the problems but requires still some work.

**Comment by aaltat** [comment]
I think I did fix all the errors and I am happy with the text. Could someone perform a review?

**Comment by just-be-dev** [comment]
LGTM 👍

**Comment by emanlove** [comment]
That formatting looks like markdown and not ReST or .rst...

**Comment by just-be-dev** [comment]
See above conversation

**Comment by aaltat** [comment]
Yes, it is markdown, changed title accordingly. Is it a problem? I tend to favor now days markdown over rst

**Comment by emanlove** [comment]
Read it. The extension for markdown is .md last time I checked.  Also all are other documentation is in proper rst formatting. Just though that was confusing...

**Pull request merged**